CONTRIBUTING.md

```diff
@@ -44,3 +46,7 @@ If you find a bug or have a question, feel free to create a new GitHub issue. Or
 We write tests primarily in groovy (with the exception of Sample stories mentioned below). The tests are running against a mock aplication located in the `jbehave-support-core-test` folder. To start it run the `jbehave-support-core-test/jbehave-support-core-test-app/src/main/java/org/jbehavesupport/core/test/app/JbehaveSupportCoreTestApplication.class`.
 
 Sample stories (`jbehave-support-core/src/test/groovy/org/jbehavesupport/test/sample`) are samples for users and tests at the same time.
+
+>
```

Comment:
Can you maybe add some header to the release section just to keep it a bit more separated?